pages/learn/develop/container-contracts.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Container Contracts
+excerpt: A guide to using container contracts to enforce device compatibility
+---
+
+# Container contracts
+
+__Note__ Container contracts are available for devices running Supervisor versions >= 10.6.0. All prior versions will not enforce any contracts.
+
+Container contracts are used to ensure the compatibility of a device to run an application service. For example, container contracts may be used to ensure that the host operating system is running a specific version of balenaOS or Supervisor such that if an application is dependent on a specific version, it only gets deployed to those devices that can meet these requirements to fulfill the contract. Currently, you may define contract requirements based on the versions of [balenaOS](/reference/OS/overview/2.x/) and [Supervisor](/reference/supervisor/supervisor-api/).
```

Comment:
```suggestion
Container contracts are used to ensure the compatibility of a device to run an individual service of an application. For example, container contracts may be used to ensure that the host operating system is running a specific version of balenaOS or balena-supervisor such that if an application is dependent on a specific version, it only gets deployed to those devices that can meet these requirements to fulfill the contract. Currently, you may define contract requirements based on the versions of [balenaOS](/reference/OS/overview/2.x/) and [balena-supervisor](/reference/supervisor/supervisor-api/).
```

I'm not sure on these, and if you think the original is better I don't really mind :)